.github/workflows/ci.yml

```diff
@@ -86,6 +89,10 @@ jobs:
         python-version: ${{ matrix.python }}
         architecture: ${{ matrix.arch }}
 
+    - name: Update CMake
+      uses: jwlawson/actions-setup-cmake@v1.3
+
+
```

Comment:
I can actually drop this - it was for PyPy support (new in CMake 3.18), but due to the SOABI being [completely broken](https://gitlab.kitware.com/cmake/cmake/-/issues/21070) at the moment, PyPy will need to wait till CMake 3.19.